content/applications/general/iot/config/updating_iot.rst

```diff
@@ -73,6 +73,81 @@ Lastly, click :guilabel:`Flash`, and wait for the process to finish.
+   for the latest update use the following URL: `https://nightly.odoo.com/16.0/nightly/windows/`. It
+   is imperative that the `16.0` be replaced with the version of Odoo that the database is running
+   on (only full versions or long supported versions). The latest file will have `latest` in its
+   filename. Note the date of upload to the right of the filename.
+
+Uninstalling Windows IoT
+------------------------
+
+Prior to upgrading the Windows virtual :abbr:`IoT (Internet of Things)` box, the previous version
+should be uninstall first. Uninstalling the Windows virtual :abbr:`IoT (Internet of Things)` box is
+done through the Windows program manager. Using any Windows version, search for `program`. Then,
+from the control panel, select :guilabel:`Add or Remove Programs`. Then, search for `Odoo` and click
+the :guilabel:`.. (three dot)` (ellipsis) menu to uninstall.
+
+Confirm the un-installation and follow the steps to uninstall through the Odoo uninstall wizard.
+
+Download and re-installation
+----------------------------
+
+To begin the re-installation, navigate to the latest installation package for Enterprise or
+Community - Windows edition at  `Odoo's nightly download page
+<https://nightly.odoo.com/16.0/nightly/windows/>`_.
+
+.. important::
+   It is imperative that the `16.0` be replaced with the version of Odoo that the database is
+   running on. The latest file will have `latest` in its filename.
+
+Next, install and setup the Odoo :file:`.exe` file. After the instructions screen, click
+:guilabel:`Next` to start the installation and agree to the :abbr:`TOS (Terms of Service)`.
+
+During the next step of the re-installation, select :guilabel:`Odoo IoT` from the :guilabel:`Select
+the type of install` drop-down menu.
+
+.. example::
+   For reference, the following should be installed:
+
+   - **Odoo server**
+   - **Odoo IoT**
+   - **Nginx WebServer**
+   - **Ghostscript interpreter**
+
+Ensure there is enough space on the computer for the installation, and click :guilabel:`Next`.
+
+Set the destination and complete the installation
+-------------------------------------------------
+
+To complete the re-installation, select the :guilabel:`Destination Folder` and click
+:guilabel:`Install`.
+
+.. warning::
+   Choosing ``C:\odoo`` as the install location allows for the Nginx server to start. Odoo's Windows
+   virtual :abbr:`IoT (Internet of Things)` box software should not be installed inside any of the
+   Window's User's directories. Doing so does **not** allow for Nginx to initialize.
+
+The installation may take a few minutes. When complete, click :guilabel:`Next` to continue.
+
+Ensure that the :guilabel:`Start Odoo` box is checked and click :guilabel:`Finish`. After
+installation, the Odoo server runs and automatically opens `http://localhost:8069` on a web browser.
+The webpage should display the :abbr:`IoT (Internet of Things)` box homepage.
+
+.. seealso::
```

Comment:
is seealso the appropriate admonition block for this kind of information? What do you think?

Reply:
Changed to tip